features/step_definitions/xmlrpc_common.rb

```diff
@@ -259,18 +259,6 @@
   detail = virtualhostmanager.getDetail(label)
 end
 
-Then(/^I should get two modules$/) do
```

Comment:
are this not used? then good to remove them! good catch

Reply:
Please double-check that...

Reply:
ok they are not used +1